File: Samples-Xamarin/Adapters/SideDrawerExamples.cs
using System;
using Java.Util;
using Com.Telerik.Android.Common;

namespace Samples
{
	public class SideDrawerExamples : Java.Lang.Object, ExamplesProvider {

		private LinkedHashMap sideDrawerExamples;

		public SideDrawerExamples(){
			this.sideDrawerExamples = this.getSideDrawerExamples();
		}

		public String ControlName() {
			return "Side Drawer";
		}

		public LinkedHashMap Examples(){
			return this.sideDrawerExamples;
		}

		private LinkedHashMap getSideDrawerExamples(){
			LinkedHashMap sideDrawerExamples = new LinkedHashMap();
			ArrayList result = new ArrayList();

			result.Add (new DrawerInitialSetupFragment());

			sideDrawerExamples.Put ("Init", result);

			return sideDrawerExamples;
		}
	}
}

File: Samples-Java/Samples/src/main/java/activities/SideDrawerExamples.java
package activities;

import java.util.ArrayList;
import java.util.LinkedHashMap;

import fragments.sidedrawer.SideDrawerFeaturesFragment;

public class SideDrawerExamples implements ExamplesProvider {
    @Override
    public String controlName() {
        return "Side Drawer";
    }

    @Override
    public LinkedHashMap<String, ArrayList<ExampleFragment>> examples() {
        LinkedHashMap<String, ArrayList<ExampleFragment>> drawerExamples = new LinkedHashMap<String, ArrayList<ExampleFragment>>();

        ArrayList<ExampleFragment> result = new ArrayList<ExampleFragment>();
        result.add(new SideDrawerFeaturesFragment());
        drawerExamples.put("Init", result);


        return drawerExamples;
    }
}
